test/search-module.ts

```diff
@@ -22,8 +22,14 @@ describe('search-module', () => {
         describe(lockFile, () => {
           before(async () => {
             await createTempDir();
-            await fs.copy(path.resolve(__dirname, 'fixture', 'multi-level-workspace'), baseDir);
-            await fs.ensureFile(path.join(baseDir, lockFile));
+            await fs.promises.cp(path.resolve(import.meta.dirname, 'fixture', 'multi-level-workspace'), baseDir, { recursive: true, force: true });
+
+            const lockfilePath = path.join(baseDir, lockFile);
+
+            if(!fs.existsSync(lockfilePath)) {
+              await fs.promises.mkdir(baseDir, { recursive: true });
+              await fs.promises.writeFile(lockfilePath, Buffer.from([]), {});
```

Comment:
Since this is a `graceful-fs` call, it'll need to be passed through `util.promisify`